### ISSUE TITLE
Rswift 7 using generators with SPM

### ISSUE THREAD

**Issue by muratyilmaz** [mac-cain13/R.swift#796]
Can I use specific generators with RswiftGeneratePublicResources/RswiftGenerateInternalResources?

**Comment by tomlokhorst** [comment]
Unfortunately no. When using these plugins, all generators are always used.
If you want to only use specific generators, you'll need to use the command line version.

**Comment by muratyilmaz** [comment]
Thank you for your answer.
If I use RswiftGenerateResourcesCommand from Xcode, the generators work. But, with RswiftModifyXcodePackages, it does not. Could you let me know if you meant this?

<img width="701" alt="Screen Shot 2022-12-27 at 20 03 50" src="https://user-images.githubusercontent.com/1283764/209698678-6b93e5b9-3075-4c81-bd13-a0501869127d.png">

**Comment by tomlokhorst** [comment]
Oh right, I forgot about the RswiftGenerateResourcesCommand 😆
Yes, you can use that, it should work correctly. You will need to run that again, each time you add or remove resources.

RswiftModifyXcodePackages is only meant to be run once, when adding a Build Tool Plug-in.
It modifies the Xcode project, it does not itself run R.swift.

**Comment by muratyilmaz** [comment]
Yep, like you said I need to run that again each time I add resources, and also I need to specify generators as arguments. Is there any way to automate this?

**Comment by muratyilmaz** [comment]
Another question is that you said you can use commandline version. How can I use it with SPM? Do I need to add it to build phase?

**Comment by anatoliykant** [comment]
@tomlokhorst 
Hey, if you need more information, I'm ready. 🙂
